src/result_processor.h

```diff
@@ -215,6 +222,8 @@ void SortAscMap_Dump(uint64_t v, size_t n);
 ResultProcessor *RPSorter_NewByFields(size_t maxresults, const RLookupKey **keys, size_t nkeys,
                                       uint64_t ascendingMap);
 
+void RPSoter_addLoadKeys(ResultProcessor *RPSorter, const RLookupKey **loadKeys, size_t nLoadKeys);
```

Comment:
I thought it would be confusing to pass 2 pairs of the same type (`RLookupKey **` and `size_t`), and a hard-to-find bug if someone accidentally switches between them

Reply:
unified the functions and added assert that nkeys should be equal or greater than nloadkeys